Update UI screenshots in Pipeline documentation #7526 (#7642)
jenkins-infra/jenkins.io: content/doc/book/pipeline/shared-libraries.adoc

> the use-case. _Manage Jenkins » System » Global Pipeline Libraries_
> as many libraries as necessary can be configured.

Reviewer: ```suggestion
the use-case. 
Navigate to _Manage Jenkins » System » Global Trusted Pipeline Libraries_ to configure as many libraries as necessary.
```
Suggestion for text update for grammar and to match the new UI
